packages/design-system-dashboard-cli/src/find-ds-components.js

```diff
@@ -163,18 +166,30 @@ function findComponents(searchStrings) {
   );
 
   const wcTagRegex = /<(va-[^\s>]+)/gms;
-  const vwWebComponents = search(vwModules, wcTagRegex);
-  const contentBuildWC = search(contentTemplates, wcTagRegex);
+  const wcUswds3Regex = /<(Va[^\s]+|va-[^\s]+)(\s|\n)[^>]*?uswds/gms;
 
-  const vwComponents = [
-    ...usedReactComponents,
-    ...vwWebComponents,
-    ...usedReactBindings,
-  ];
+  let vwWebComponents;
+  let contentBuildWC;
+  let vwComponents;
 
-  const data = tallyResults(vwComponents, contentBuildWC);
+  if (searchStrings?.includes('uswds')) {
+    vwWebComponents = search(vwModules, wcUswds3Regex);
+    contentBuildWC = search(contentTemplates, wcUswds3Regex);
+    vwComponents = [...vwWebComponents];
+  } else {
+    vwWebComponents = search(vwModules, wcTagRegex);
+    contentBuildWC = search(contentTemplates, wcTagRegex);
+    vwComponents = [
+      ...usedReactComponents,
+      ...vwWebComponents,
+      ...usedReactBindings,
+    ];
+  }
 
-  return filterSearchedComponents(data, searchStrings);
+  const data = tallyResults(vwComponents, contentBuildWC);
+  return searchStrings?.includes('uswds')
```

Comment:
`filterSearchedComponents` expects the name of a component, not the name of a prop. Passing `uswds` to it returns nothing because there is no uswds component. Instead, the data is just returned directly.